### PR TITLE
Use commons-ui library 

### PR DESCRIPTION
**Pull request by ChamseddineBhd**
Signed-off-by: chamseddine <chamseddine.benhamed@ensi-uma.tn>

### CODE REVIEW

**Comment by geofjamg** [comment]
When running the app, I have a mix of light/dark theme with the toolbar and the app:
image

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 name: CI
 
-on: [push]
+on:
```
Comment (geofjamg) [inline .github/workflows/build.yml:3]: To revert?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -12,11 +13,25 @@ jobs:
         uses: actions/checkout@v2 # If you're using actions/checkout@v2 you must set persist-credentials to false in most cases for the deployment to work correctly.
         with:
           persist-credentials: false
+          path: study-app
```
Comment (geofjamg) [inline .github/workflows/build.yml:16]: This is necessary?

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -12,11 +13,25 @@ jobs:
...
+            node-version: ${{ matrix.node-version }}
+            registry-url: https://npm.pkg.github.com/
+            scope: '@gridsuite'
 
       - name: Install and Build
         env:
           CI: false
+          NODE_AUTH_TOKEN: ${{secrets.GITHUB_TOKEN}}
         run: |
+          cd $GITHUB_WORKSPACE/study-app
```
Comment (geofjamg) [inline .github/workflows/build.yml:34]: Not necessary?

File: README.md [hunk 4]
```diff
@@ -1 +1,18 @@
 # study-app
+
+This app is using commons-ui library which released in the github packages.
+
+To run the app you should first connect to github packages registry
+
+  > npm login --registry=https://npm.pkg.github.com
```
Comment (geofjamg) [inline README.md:7]: But we don't need to login to just get commons-ui dependency?

Comment (ChamseddineBhd) [reply README.md:7]: we need to login the github packages firs!

**Comment by jonenst** [comment]
"chmits-common-ui" slipped in package-lock.json

**Pull request merged**